tests/python/contrib/test_ethosu/test_legalize.py

```diff
@@ -881,7 +888,7 @@ def verify(ext_func):
         ([1, 4, 4], [4, 1], False),
     ],
 )
-@pytest.mark.parametrize("activation_function", ["NONE", "RELU"])
+@pytest.mark.parametrize("activation_function", [None, tf.nn.relu, tf.nn.relu6, relu_n1_to_1])
```

Comment:
```suggestion
@pytest.mark.parametrize("activation_function", [None, tf.nn.relu])
```
Since this test looks at the graph entirely at Relay level, where all the RELUs are just Relay clip operations, I don't think we benefit much from extra 70 tests with just different min and max attributes to clip.

Reply:
I agree with this, I will add separate test for case with MAX operation and relu_n1_to_1 activation.